Imagetospeech/image_to_text.py

```diff
@@ -0,0 +1,15 @@
+import pytesseract
+from PIL import Image
+
+
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract'
+
```

Comment:
@Amit366 take the input of `pytesseract.pytesseract.tesseract_cmd` from the user, since the adress of pytesseract file can be different for differen user.